skyportal/facility_apis/generic.py

```diff
@@ -127,6 +129,13 @@ def create_target_text(request, request_type='email'):
         f"Username: {target['username']}"
     )
 
+    if comments:
```

Comment:
maybe check that the obj has comments too, to not have `Comments:` empty when there is no comments, but just skip it or say there is no comments.